js/test/root/testlocalematch.js

```diff
@@ -1106,6 +1106,39 @@ module.exports.testlocalematch = {
         test.equal(locale.getSpec(), "ka-Geor-GE");
         test.done();
     },
+    testLocaleMatcherGetLikelyLocaleByLocaleCodehy: function(test) {
+        test.expect(3);
+        var lm = new LocaleMatcher({
+            locale: "hy"
+        });
+        test.ok(typeof(lm) !== "undefined");
+        var locale = lm.getLikelyLocale();
+        test.ok(typeof(locale) !== "undefined");
+        test.equal(locale.getSpec(), "hy-Armn-AM");
+        test.done();
+    },
+    testLocaleMatcherGetLikelyLocaleByLocaleCodehy2: function(test) {
+        test.expect(3);
+        var lm = new LocaleMatcher({
+            locale: "AM"
+        });
+        test.ok(typeof(lm) !== "undefined");
+        var locale = lm.getLikelyLocale();
+        test.ok(typeof(locale) !== "undefined");
+        test.equal(locale.getSpec(), "hy-Armn-AM");
+        test.done();
+    },
+    testLocaleMatcherGetLikelyLocaleByLocaleCodehy3: function(test) {
+        test.expect(3);
+        var lm = new LocaleMatcher({
+            locale: "Aram-AM"
```

Comment:
Armn? Aram is not a valid script code...

Reply:
I fixed it